doc/jemalloc.xml.in

```diff
@@ -1055,7 +1055,9 @@ mallctl("arena." STRINGIFY(MALLCTL_ARENAS_ALL) ".decay",
         linkend="arena.i.dirty_decay_ms"><mallctl>arena.&lt;i&gt;.dirty_decay_ms</mallctl></link>
         for related dynamic control options.  See <link
         linkend="opt.muzzy_decay_ms"><mallctl>opt.muzzy_decay_ms</mallctl></link>
-        for a description of muzzy pages.</para></listitem>
+        for a description of muzzy pages.  Note that when the huge_threshold
+        feature is enabled, the special auto arenas may use its own decay
+        settings.</para></listitem>
```

Comment:
@davidtgoldblatt: added some note here. I noticed that extent_hooks already mention about auto arenas may have `created extents prior to the application having an opportunity to take over extent allocation` http://jemalloc.net/jemalloc.3.html#arena.i.extent_hooks, and here it's not directly related to the hooks so didn't touch that part.

Reply:
I might even be a little more obfuscatory, such as by saying on the extent hook setter mallctl something like "note that API guarantees may be relaxed on automatically created arenas; hooks set there may be treated as hints, or called in a 'best effort' fashion".

Not dead-set on this; feel free to re-word before merging or disregarding if you disagree.

Reply:
This sounds good. I'll include it in the last commit.